### PR TITLE
Update the whoami command to follow the VIP-CLI style guide

### PR DESCRIPTION
**Pull request by terriann**
## Description

Updated the voice, tense, and format of command descriptions, examples, and usage models for the `vip whoami` command to follow the VIP-CLI style guide.

## Pull request checklist

- [ ] Update [SETUP.md](https://github.com/Automattic/vip-cli/blob/trunk/docs/SETUP.md#list-of-environmental-variables) with any new environmental variables.
- [ ] Update [the documentation](https://github.com/Automattic/vip-cli/blob/trunk/docs).
- [x] [Manually test](https://github.com/Automattic/vip-cli/blob/trunk/docs/TESTING.md#manual-testing) the relevant changes.
- [x] Follow the [pull request checklist](https://github.com/Automattic/vip-cli/blob/trunk/docs/RELEASING.md#new-pull-requests)
- [ ] Add/update [automated tests](https://github.com/Automattic/vip-cli/blob/trunk/docs/TESTING.md#automated-testing) as needed.

## New release checklist

- [ ] [Automated tests](https://github.com/Automattic/vip-cli/blob/trunk/docs/TESTING.md#automated-testing) pass.
- [ ] The [Preparing for release checklist](https://github.com/Automattic/vip-cli/blob/trunk/docs/RELEASING.md#preparing-for-release) is completed.

## Steps to Test

Observe the results of the `--help` display for impacted commands:

- `vip whoami`

### CODE REVIEW

File: src/bin/vip-whoami.ts [hunk 1]
```diff
@@ -6,6 +6,9 @@ import command from '../lib/cli/command';
 import * as exit from '../lib/cli/exit';
 import { trackEvent } from '../lib/tracker';
 
+const baseUsage = 'vip whoami';
+const exampleUsage = 'vip @example-app.develop whoami';
```
Comment (yolih) [inline src/bin/vip-whoami.ts]: `whoami` is one of the very few commands that does not require an app alias. 
I suggest that we update the command example to instead be the simpler form:
`vip whoami`
I just tested this command with a non VIP-team user, and I think it would be helpful for the example to include the whole shebang:
```
$ vip whoami
- Howdy user@example.com!
- Your user ID is 1234
```
I am trying my best to ignore the "Howdy" and fight the urge to let this scope creep....

Comment (terriann) [reply src/bin/vip-whoami.ts]: That makes sense - I added the exception about application.env aliases to the style guide.

Comment (yolih) [reply src/bin/vip-whoami.ts]: Perfect. Thank you!

**Review by yolih — APPROVED** [review]
This looks good to me!

**Review by WPprodigy — APPROVED** [review]
